aten/src/ATen/native/native_functions.yaml

```diff
@@ -1116,12 +1116,11 @@
 - func: empty.out(int[] size, *, MemoryFormat? memory_format=None, Tensor(a!) out) -> Tensor(a!)
   device_guard: False
 
-- func: empty_like(Tensor self) -> Tensor
-  use_c10_dispatcher: full
+- func: empty_like(Tensor self, *, MemoryFormat? memory_format=None) -> Tensor
```

Comment:
With the benefit of hindsight, I should have asked: why do we have two overloads of `empty_like` in the first place?!

Reply:
Do you mean this line and line 1123?

Reply:
yes. But see my most recent PR.

Reply:
As far as I remember combining them together was never considered by me.